provider/uprops/src/upropdump_serde.rs

```diff
@@ -23,5 +23,30 @@ pub mod binary {
 }
 
 pub mod enumerated {
-    // TODO(#883)
+    #[derive(serde::Deserialize)]
+    pub struct EnumeratedPropertyMapRange {
+        pub start: u32,
+        pub end: u32,
+        pub name_idx: u32,
+        pub long_name: String,
+        pub short_name: String,
```

Comment:
`question(n-b)`: Can we use `TinyStr` here and in `EnumeratedPropertyMap`? Would it be useful? I'm not sure if this is the structure that we rely on in production or just a feeder data?

Reply:
This is the "feeder data" struct, although I'm not opposed to using TinyStr here anyway.

Reply:
Note also that a non-trivial number of long names are >16 characters ("Close_Punctuation", "Canadian_Aboriginal", etc...).